test-suite/Makefile

```
@@ -148,8 +158,29 @@ PREREQUISITELOG = $(addsuffix .log,$(wildcard prerequisite/*.v)) .csdp.cache
 .DELETE_ON_ERROR:
 .PHONY: all run clean $(SUBSYSTEMS)
 
+ifeq ($(COQ_NOT_CONFIGURED),true)
+all:
+	@echo ""
+	@echo "Coq has not been configured, do any of: "
```

```suggestion
	@echo "Coq has not been configured, please run one of the following: "
```
We only want one to be run right?